Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM golang:1.24.4-alpine AS builder
+
+ARG TARGETARCH
+ARG TARGETOS
+
+ENV GOPATH /
+
+COPY go.mod .
+COPY go.sum .
+RUN go mod download && go mod verify
+COPY . .
+
+RUN mkdir /app
+RUN GOOS=$TARGETOS GOARCH=$TARGETARCH go build -o /app/bin .
+
+# alpine
+FROM alpine:3.22.0
```

Comment:
Nitpick: I prefer distroless (e.g. FROM gcr.io/distroless/static-debian12), it's smaller (~ 2MB) and already configures a nonroot user and doesn't have performance issues in the memory allocator (musl libc it's allocator can be slow). It doesn't have curl so it would require removing the HEALTHCHECK.

Reply:
Can we install curl on it, like we do here? It's really nice to have a health check when running in docker (especially for local development), and I sometimes use it to debug something in a running cluster (although that's probably a hack..)

Reply:
I guess this is a decent solution: https://github.com/GoogleContainerTools/distroless/issues/183#issuecomment-571723446 (copies `wget` from busybox)

Reply:
I think using `alpine` for now is fine. Copying wget from busybox is also an option, but I wouldn't want to do it through `ADD https://`